app/views/items/index.html.erb

```diff
@@ -4,5 +4,6 @@
     <li>Description: <%= item.description %><br/></li>
     <li>Price: $<%= item.price %><br/></li>
     <%= image_tag src="#{item.image}" %>
+    <%= button_to "Add to Cart", carts_path(item_id: item.id) , class: "btn btn-danger" %>
```

Comment:
btn and btn-danger...
I'm not opposed to throwing in html classes in preparation for styling, but would rather have this in a different branch and would rather talk about styling as a group first